### PR TITLE
Update EIP-5484: Move to Last Call

### PR DESCRIPTION
**Pull request by buzzcai**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Review by Pandapip1 — COMMENTED** [review]
There's a typo on line 83–there's an extra space.

**Comment by buzzcai** [comment]
@Pandapip1 Markdown Linter is failing with "Error: Failed due to error: Error: ENOENT: no such file or directory, open 'config/.markdownlint.yaml'" while .markdownlint.yaml is in config directory last time I checked.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
The security considerations section needs to be completed.

**Comment by buzzcai** [comment]
@Pandapip1 done

File: EIPS/eip-5484.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ title: Consensual Soulbound Tokens
 description: Interface for special NFTS with immutable ownership and pre-determined immutable burn authorization
```
Comment (Pandapip1) [inline EIPS/eip-5484.md:4]:
```suggestion
description: Interface for special NFTs with immutable ownership and pre-determined immutable burn authorization
```

**Comment by buzzcai** [comment]
> There's a newline missing between L74 and L75.
> 
> On L76, there's a minor grammatical eror.

@Pandapip1 do you mind pointing out the grammatical error on L76? (now L77)

**Review by Pandapip1 — APPROVED** [review]

**Pull request merged**